modules/navier_stokes/include/fvkernels/INSFVMomentumAdvection.h

```diff
@@ -25,52 +26,23 @@ class INSFVPressureVariable;
  * An advection kernel that implements interpolation schemes specific to Navier-Stokes flow
  * physics
  */
-class INSFVMomentumAdvection : public FVMatAdvection
+class INSFVMomentumAdvection : public FVMatAdvection, UserObjectInterface
```

Comment:
This should be moved farther up the inheritance chain, e.g. to `FVKernel` probably